schemas/observation/digital-signature.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var Joi = require( 'joi' );
+
+var signatureInfo = {
+	'sign'    : Joi.string().optional().description( 'string that the user inputs for digital signatures' ),
+	'date'    : Joi.date().optional().description( 'The date of which the user signed the observation' ),
```

Comment:
You're probably going to want to type this as `Joi.date().timestamp()...`. We'll want to ensure time is included with the date.